homeassistant/components/zwave_js/__init__.py

```diff
@@ -339,6 +339,14 @@ async def async_on_node_added(self, node: ZwaveNode) -> None:
                 node,
             )
 
+        # Create statistics sensors for each device
+        await self.driver_events.async_setup_platform(Platform.SENSOR)
```

Comment:
Why call this again? It was already set up `if not node.is_controller_node`

Reply:
it's needed for controllers too but I take your point